File: tests/fixtures/code/Sample.java
public class Sample {
    static void greet(String name) {
        System.out.println("Hello, " + name);
    }

    static int compute(int n) {
        int total = 0;
        for (int i = 0; i < n; i++) {
            if (i % 2 == 0) {
                total += i;
            } else {
                total += i * 2;
            }
        }
        return total;
    }

    public static void main(String[] args) {
        greet("world");

        for (int i = 0; i < 3; i++) {
            System.out.println("i: " + i);
        }

        int value = compute(10);
        System.out.println("value: " + value);
    }
}

File: tests/fixtures/code/sample.js
function greet(name) {
  console.log(`Hello, ${name}`);
}

function compute(n) {
  let total = 0;
  for (let i = 0; i < n; i++) {
    if (i % 2 === 0) {
      total += i;
    } else {
      total += i * 2;
    }
  }
  return total;
}

function main() {
  greet("world");

  for (let i = 0; i < 3; i++) {
    console.log("i:", i);
  }

  const value = compute(10);
  console.log("value:", value);
}

main();

